tests/test_utils.py

```diff
@@ -295,7 +295,7 @@ def test_check_status_code_for_missing_status_code(
     captured = capsys.readouterr()
 
     if verbose:
-        assert captured.out == "Content received from server:\nForbidden\n"
+        assert "Content received from server:\nForbidden\n" in captured.out
     else:
         assert captured.out == "NOTE: Try --verbose to see response content.\n"
```

Comment:
Why this change? It seems unrelated to the new log message, and it's inconsistent with the other assertion.

Reply:
I'm running into an issue where previous log messages are being captured by `capsys`. I tried using `with capsys.disabled()` to temporarily stop the capture but that doesn't seem to work, possible due to the use of `tox`. Another option is to read the output to a temporary variable before the output we care about to isolate the needed output and make sure its exactly equal to what we expect. Have you resolved an issue like this in `twine` before?

Reply:
The previous log messages are indeed confusing, but ultimately misleading. The issue was that since `make_settings(verbose=True)` ultimately calls `get_config`, the new log message was being added to the output.

So, in `test_check_status_code_for_missing_status_code`, the error was:

```
>           assert captured.out == "Content received from server:\nForbidden\n"
E           AssertionError: assert 'Using config...\nForbidden\n' == 'Content rece...\nForbidden\n'
E             + Using configuration from /private/var/folders/mz/3_sz949s3yvg42z3twf9y56r0000gn/T/pytest-of-brian/pytest-11/test_check_status_code_for_mis0/.pypirc
E               Content received from server:
E               Forbidden
```

Note the `+ Using` line. In the new `test_get_config_log_location`, the call to `utils.get_config` was redundant, causing the the log message to be printed twice.

I pushed https://github.com/pypa/twine/pull/675/commits/d4112af35d85e2ba848e95e873bf04ab0754927b to tweak the tests to account for that.